lib/applications/resolvers.ts

```diff
@@ -857,27 +857,109 @@ export const createReplacementApplication: Resolver<
   MutationCreateReplacementApplicationArgs,
   CreateReplacementApplicationResult
 > = async (_, args, { prisma }) => {
-  // TODO: Validation
   const { input } = args;
+
   const {
```

Comment:
To avoid having to manually pass in all these fields below, we can destructure fields from `input` two times:
```typescript
const {
  applicantId,
  ...originalfields,
  ...data
} = input
const {
  ...new fields that need to be validated
} = input
```

This way, the `prisma` call below should remain unchanged